entities/status.py

```diff
@@ -0,0 +1,6 @@
+from enum import Enum
+
+
+class Status(Enum):
```

Comment:
Consider changing it from verbal string status to standard http statuses (for instance 200 for success) so it'll be easy to use the API (consider the case that someone new is trying to call our API and doesn't understand why it's response status is not 200)